README.md

```diff
@@ -4,6 +4,14 @@
 
 **tuftool** is a Rust command-line utility for generating and signing TUF repositories.
 
+## ❗ Warnings
+### Windows
+- Tests *will* break under windows if ; before cloning ; you don't set the git flag `core.autocrlf = input`. This is due to the fact that some tests require some test files to have a *precise* byte size and that `core.autocrlf` makes some files bigger while converting from LF to CLRF.
+- The build process requires [Cygwin](https://cygwin.com/install.html) to work properly.
+- Some integration tests require [Docker Desktop](https://www.docker.com/products/docker-desktop) to work properly.
+### UNIX/Linux
+- Integration tests require `docker` to work.
+
```

Comment:
I think the requirement for `docker` should be clear enough for users of any platform.

I think the idea of warning Windows developers about the potential for line ending changes to cause failing tests is a good one. I changed the working to say that we have mitigated it with a .gitattributes file (as suggested above), but I think the warning is still valid because I could see how the issue could pop-up somewhere else over time.

```suggestion
## Integration Testing

Integration tests require `docker`.

### Windows❗ Warning

Tests can break on Windows if Git's `autocrlf` feature changes line endings.
This is due to the fact that some tests require files to have a *precise* byte size and hash signature.
We have mitigated this with a `.gitattributes` file in the test data directory.
```

Reply:
I'll edit that, but I think that still warning users that cygwin is required for integration tests to run is important ; because we are still using `run.sh` and `teardown.sh` for integration tests and installing only make for windows wouldn't work because of the .sh files that need a bash-like environment to run.

Reply:
I moved this info into a comment in the `Cargo.toml` file, under the `integ` feature comment.

Reply:
Sorry I removed the cygwin statement from this readme suggestion by mistake. It can be added back to the Windows warning here.

Reply:
Oh, no problem, putting it back in https://github.com/awslabs/tough/pull/342/commits/cad8d347008ffaf834caf270177b617052f9c94a